lib/app-components/addon/components/branded-navbar/template.hbs

```diff
@@ -11,7 +11,6 @@
             <div class='navbar-header'>
                 <LinkTo 
                     @route={{@brandRoute}}
-                    @provider={{this.theme.provider}}
```

Comment:
@futa-ikeda Is this going to cause any problems for your branding stuff? I'm personally okay with links not being colored for a provider, but I'm not sure if that's actually happening with this.

Reply:
Yeah this is quite confusing because I don't think passing a `@provider` argument to a `<LinkTo>` object would do anything because it's a native Ember component that doesn't know what a `@provider` is.

Reply:
Right, not an `OsfLink`. Good point.